assets/src/scss/layout/_block-classes.scss

```diff
@@ -0,0 +1,45 @@
+.wp-block-query-pagination {
+  a {
+    color: black;
+    font-weight: bold;
+  }
+
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.page-numbers {
+  &:not(:last-child) {
+    margin-inline-end: 8px;
+  }
+
+  &.current {
+    border: 1px solid black;
+    border-radius: 4px;
+    padding: 8px 12px;
+  }
+}
+
+.wp-block-post-terms {
+  font-weight: 500;
+}
+
+.taxonomy-post_tag a:before {
```

Comment:
This doesn't seem to work as expected on RTL sites:
<img width="923" alt="Screenshot 2022-04-12 at 16 29 31" src="https://user-images.githubusercontent.com/6949075/162985813-20d940ea-93e0-40c4-8bf9-14f340a598d5.png">

Reply:
Nice catch, I fixed it with `display: inline-block`, but still don't see why it would do something differently for 1 of the tags :exploding_head: afaik they have the same CSS.